app/main.py

```diff
@@ -1 +1,41 @@
-# write your code here
+import os
+import shutil
+
+
+def move_file(command: str) -> None:
+    parts = command.split(" ")
+    if len(parts) != 3 or parts[0] != "mv":
+        raise ValueError("Invalid command")
+
+    source_file = parts[1]
+    destination = " ".join(parts[2:])
```

Comment:
You can do this in `split()` function 
```suggestion
     cmd, source_file, destination = command.split()
```